<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,24">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,1,1,6">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnBend">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>CAM工序规划</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/img/Bend.ico</normaloff>:/img/Bend.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnSelect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>设备选型</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/img/Select.ico</normaloff>:/img/Select.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnSetting">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>工艺规划</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/img/Setting.ico</normaloff>:/img/Setting.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnDiaplay">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>三维展示</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/img/Display.ico</normaloff>:/img/Display.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_7" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnProg">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>数据库管理</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/img/Program.ico</normaloff>:/img/Program.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnShow">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <property name="text">
             <string>工件显示</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/img/Show.ico</normaloff>:/img/Show.ico</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="autoRepeatDelay">
             <number>300</number>
            </property>
            <property name="popupMode">
             <enum>QToolButton::DelayedPopup</enum>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextUnderIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_9" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_data" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:0.035, y2:0, stop:0.427861 rgba(206, 206, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,25">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labFilename">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>文件名：</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="childwidget" native="true">
      <property name="font">
       <font>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_O"/>
    <addaction name="action_S"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_up"/>
    <addaction name="action_down"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>小玩意</string>
    </property>
    <addaction name="actionT"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_V"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_O"/>
   <addaction name="separator"/>
   <addaction name="action_S"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="action_up"/>
   <addaction name="action_down"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_O">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/open.png</normaloff>:/img/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>保存文件(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存机器人代码(S)</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/Machine.png</normaloff>:/img/Machine.png</iconset>
   </property>
   <property name="text">
    <string>显示设备</string>
   </property>
  </action>
  <action name="action_up">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/zoom-in.png</normaloff>:/img/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>放大视图</string>
   </property>
   <property name="toolTip">
    <string>放大视图</string>
   </property>
  </action>
  <action name="action_down">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/zoom-out.png</normaloff>:/img/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>缩小视图</string>
   </property>
   <property name="toolTip">
    <string>缩小试图</string>
   </property>
  </action>
  <action name="actionT">
   <property name="text">
    <string>定时器</string>
   </property>
   <property name="toolTip">
    <string>定时器</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
